content-src/components/CollapsibleSection/CollapsibleSection.jsx

```diff
@@ -172,9 +179,7 @@ export class _CollapsibleSection extends React.PureComponent {
               <span className="learn-more-link-wrapper">
                 {learnMore &&
                   <span className="learn-more-link">
-                    <a href={learnMore.link.href}>
-                      <FormattedMessage id={learnMore.link.id} />
-                    </a>
+                    <a href={learnMore.link.href} data-l10n-id={learnMore.link.id}>{learnMore.link.text}</a>
```

Comment:
This comes from section definitions:
https://github.com/mozilla/activity-stream/blob/1139bfc682ff544697ba5289a58b7623f3bc72d0/lib/SectionsManager.jsm#L34-L37

This is getting cleaned up in #5127 to take a `message` that can either be raw string or fluent id instead of two separate link properties.